application/apis/narou/type.py

```diff
@@ -68,8 +68,74 @@ def is_valid_date_for_rank_type(rank_type, date):
 
 
 class OutType(Enum):
-    """なろうランキングout."""
+    """なろうAPI関連のout."""
 
     YAML = "yaml"
     JSON = "json"
     PHP = "php"
+
+
+class NarouOrderType(Enum):
+    """なろうAPIの出力順序."""
+
+    NEW = "new"
+    FAVORITE_NOVEL_COUNT = "favnovelcnt"
+    REVIEW_COUNT = "reviewcnt"
+    HYOKA = "hyoka"
+    HYOKA_ASC = "hyokaasc"
+    DAILY_POINT = "dailypoint"
+    WEEKLY_POINT = "weeklypoint"
+    MONTHLY_POINT = "monthlypoint"
+    QUARTER_POINT = "quarterpoint"
+    YEARL_YPOINT = "yearlypoint"
+    IMPRESSION_COUNT = "impressioncnt"
+    HYOKA_COUNT = "hyokacnt"
+    HYOKA_COUNT_ASC = "hyokacntasc"
+    WEEKLY = "weekly"
+    LENGTH_DESC = "lengthdesc"
+    LENGTH_ASC = "lengthasc"
+    GENERAL_FIRST_UP = "generalfirstup"
+    NCODED_ESC = "ncodedesc"
```

Comment:
@takuron1996 
```suggestion
    NCODE_DESC = "ncodedesc"
```
のタイポかもです🙏

Reply:
@watame 
[36d7c21](https://github.com/takuron1996/narou_novel_batch/pull/11/commits/36d7c21bd3405a6a98705810b145dc9f7b522406)で対応しました。